### DISCUSSION TITLE
Migrating from v2: error handling

### DISCUSSION THREAD

**Discussion by mititch** [etsy/open-api#841] [Q&A]
Found big hole in migration guide https://developers.etsy.com/documentation/migration/index . There is nothing about update in errors format, but many integration (ours for example) relies on errors from Etsy API.

From v2 documentations https://www.etsy.com/developers/documentation/getting_started/api_basics#section_standard_response_codes 

> The X-Error-Detail HTTP header will contain a detailed description of many errors.

As I can see, in v3 that approach was removed. And now we should rely on `error` field in response body json:

![image](https://user-images.githubusercontent.com/3614464/215558049-d3e99f50-c49e-4707-b14f-eef8183adace.png)

So, the question: are messages in new `error` field still the same as were in old `X-Error-Detail` header? If not: how to map old and new error messages.

It will be cool to have this info in migration guide.

**Comment by Gareth064** [comment]
In this discussion the user says that it also returns an array of errors.

https://github.com/etsy/open-api/discussions/838

Of course we don't know if that behaviour is the same across the whole API
